compiler/rustc_ast/src/visitors.rs

```diff
@@ -381,7 +381,7 @@ macro_rules! make_ast_visitor {
             make_visit!{Const, visit_constness, walk_constness}
             make_visit!{TyAliasWhereClauses, visit_ty_alias_where_clauses, walk_ty_alias_where_clauses}
             make_visit!{Option<P<QSelf>>, visit_qself, walk_qself}
-            // TODO: Remove P! on implementers
+            // FIXME: Remove P!
```

Comment:
I tried removing this P!, but there is a chain of functions that needs to be changed and I was not confident enough to do this. So I'm changing it to FIXME